#! /usr/bin/env python

PACKAGE='mbzirc_task2_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("meanshift_sp1", double_t, 0, "", 20.0, 0.0, 400.0)
gen.add("meanshift_sp2", double_t, 0, "", 20.0, 0.0, 400.0)
gen.add("meanshift_sr1", double_t, 0, "", 20.0, 0.0, 400.0)
gen.add("meanshift_sr2", double_t, 0, "", 20.0, 0.0, 400.0)
gen.add("clustering_thre", int_t, 0, "", 1000, 0, 10000)
gen.add("clustering_mini_thre", int_t, 0, "", 200, 0, 5000)
gen.add("screen_debug", bool_t, 0, "screen_debug", True)

exit(gen.generate(PACKAGE, "mbzirc_task2_perception", "MeanshiftFiltering"))
